spec/features/reviews/user_can_create_review_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+describe "As a visitor" do
```

Comment:
doo we need a before each in case we add to this test later?

Reply:
I'm fine with either route. I figured we could just change to each if we did end up adding tests.